### ISSUE TITLE
Proposal of moving resources to a new (existing) resource module

### ISSUE THREAD

**Issue by johlju** [PowerShell/DscResources#374]
When going through the DSC Resource Kit I have seen a lot of resource modules, many with just a single resource, that could be moved to another repository. 
Moving these resource would, I think, reduce work when having fewer resource modules to maintain. I also think it would be simpler for the community discuss resources if those that fit together was in the same resource module.

Below is a suggestion how we could move this. It's open for discussion, and please comment on this if I have missed a resource, or if you see any issues with moving a particular resource.

Done | Resource | Resource(s) | Move to | Comment
--- | --- | --- | --- | ---
:heavy_check_mark: | [xPendingReboot](https://github.com/PowerShell/xPendingReboot) | xPendingReboot | ComputerManagementDsc | Discussed in https://github.com/PowerShell/xPendingReboot/issues/12.
:heavy_check_mark: | [xWinEventLog](https://github.com/PowerShell/xWinEventLog) | xWinEventLog | ComputerManagementDsc |
:heavy_check_mark: | [xPowerShellExecutionPolicy](https://github.com/PowerShell/xPowerShellExecutionPolicy) | xPowerShellExecutionPolicy | ComputerManagementDsc |
:heavy_check_mark: | [SystemLocaleDsc](https://github.com/PowerShell/SystemLocaleDsc) | SystemLocale | ComputerManagementDsc | ~~Discussed in https://github.com/PowerShell/SystemLocaleDsc/issues/14 and https://github.com/PowerShell/ComputerManagementDsc/issues/50.~~ LanguageDsc is being moved into ComputerManagementDsc (https://github.com/PowerShell/ComputerManagementDsc/pull/160) which will supersede SystemLocaleDsc. Deprecate SystemLocaleDsc once PowerShell/ComputerManagementDsc#160 is merged
In progress | [xDismFeature](https://github.com/PowerShell/xDismFeature) | xDismFeature | ComputerManagementDsc |
:heavy_check_mark: | [xRemoteDesktopAdmin](https://github.com/PowerShell/xRemoteDesktopAdmin) | xRemoteDesktopAdmin | ComputerManagementDsc | PR https://github.com/PowerShell/ComputerManagementDsc/pull/227
-- | [xWindowsRestore](https://github.com/PowerShell/xWindowsRestore) | xSystemRestore, xSystemRestorePoint | ComputerManagementDsc |
-- | [xWindowsEventForwarding](https://github.com/PowerShell/xWindowsEventForwarding) | xWEFCollector. xWEFSubscription | ComputerManagementDsc |
:heavy_check_mark: | [xTimeZone](https://github.com/PowerShell/xTimeZone) | xTimeZone | ComputerManagementDsc |
:heavy_check_mark: | [xSystemSecurity](https://github.com/PowerShell/xSystemSecurity) | xIEEsc, xUAC | ComputerManagementDsc |
:heavy_check_mark:  | [xSystemSecurity](https://github.com/PowerShell/xSystemSecurity) | xFileSystemAccessRule | FileSystemDsc (new module) |
:heavy_check_mark: | [xSmbShare](https://github.com/PowerShell/xSmbShare) | xSmbShare | ComputerManagementDsc |
-- | [xPhp](https://github.com/PowerShell/xPhp) | xPhpProvision | xWebAdministration | I thought that this resource needs IIS on Windows so that would make it fit into xWebAdministration.

**Comment by PlagueHO** [comment]
Great list @johlju - very much inline with what I would like to do.

There is a couple of other issues where moving resources into ComputerManagementDSc:
https://github.com/PowerShell/ComputerManagementDsc/issues/127
https://github.com/PowerShell/DscResources/issues/288

@ld0614 had suggested that he could add his Language Module to ComputerManagementDsc (https://github.com/ld0614/LanguageDsc), superseding SystemLocaleDsc. @ld0614, are you still interested in doing this? I think the language module contains a lot more functionality than the SystemLocaleDsc one. @Johlju - what do you think?

@ld0614 - your module looks like it has some really great functionality. Needs a few tweaks to make it HQRM, but nothing we can't get sorted easily enough.

**Comment by johlju** [comment]
@PlagueHO @Id0614 I forgot about the discussion around LanguageDsc. If LangagueDsc makes SystemLocaleDsc then I think we just deprecate SystemLocaleDsc once LanguageDsc is merged into ComputerManagementDsc (if @ld0614 is still up for it).

**Comment by ld0614** [comment]
I'm still up for merging LangaugeDSC into ComputerManagement, that last time we spoke about it, ComputerManagement was close to changing its name so I was waiting for the name change to avoid needing to rename the LanguageDSC resources just to rename them a month or two later.  If thats not really an issue or the work is taking longer than anticipated that I'm quite happy to start the merge process.

**Comment by PlagueHO** [comment]
Hi @ld0614 - the name change to ComputerManagementDsc is completed :grin: It was even released to the PSGallery. https://github.com/PowerShell/ComputerManagementDsc

**Comment by ld0614** [comment]
Whoops, I missed that one :blush: I'll do a review of the HQRM guidelines tomorrow and raise a PR to start the merge process

**Comment by PlagueHO** [comment]
Still working on these!

**Comment by johlju** [comment]
Relabeled this one to help wanted so the community know that anyone can help with these, and labeled it enhancement so the issue doesn't autoclose. Two finished so far. 😃

**Comment by johlju** [comment]
xSmbShare has been moved, and xRemoteDesktopAdmin is in progress. If someone else want to move resources, please feel free to do so. Have in mind though that it is a lot of work making the resource high quality (it is often not just moving 1:1).

**Comment by PlagueHO** [comment]
I'm planning on doing xPendingReboot - I think I stuck my hand up for that a while ago. Something I might get onto over the weekend if I can.

**Comment by PlagueHO** [comment]
Getting onto xPendingReboot now.

**Comment by PlagueHO** [comment]
@johlju - can you mark PendingReboot as done? I'll submit a PR to this repo shortly to update the resource list.

**Comment by PlagueHO** [comment]
@johlju - also, xWinEventLog has been completed. So you could update that one in the list too.

**Comment by johlju** [comment]
Done

**Comment by PlagueHO** [comment]
I'll probably move SystemLocaleDsc next (when I get time).

**Comment by johlju** [comment]
Didn’t LanguageDsc override SystemLocaleDsc? See comment column above on that.

**Comment by johlju** [comment]
Better to pick up LanguageDsc PR and move it over the finish line?

**Comment by PlagueHO** [comment]
Yes true - forgot about that one!

**Comment by Outek** [comment]
What's about this one:

https://github.com/microsoft/SChannelDsc

Should we integrate it into ComputerManagementDsc as well?

**Comment by johlju** [comment]
Might have been a good idea, but there are this module that was recently added https://github.com/dsccommunity/SChannelDsc not sure if it the one you linked to that have been moved there. 🤔

**Comment by Outek** [comment]
This seem to be the same.

**Comment by johlju** [comment]
When we have moved ComputerManagementDsc over to DSC Community we could raise the question again? Maybe there are good reasons to keep this one separate. But I could see it in ComputerManagementDsc. 🤔

**Comment by Outek** [comment]
Sure 👍

**Comment by PlagueHO** [comment]
I'm OK with moving it in here but as it is 5 resources and they're all related to security suites it might not  be a good idea. It sort of makes sense to keep the 5 of them bundled together on their own I think?

**Comment by johlju** [comment]
There might be a better idea to move it so a "ComputerSecurityDsc" or something instead together with xSystemSecurity. Or call it SystemSecurityDsc and moce SChannelDsc there too. 🤔 

The main reason to concatenate modules was to easy the release process, not needing to manually release so many modules. But since we are moving the modules over to @dsccommunity and implementing a release pipeline in each module that main reason is no longer valid. So if we choose to concatenate modules it would be because of some other reason, for example not needing to download different modules, and easier for user to find resources and documentation in less places. 🤔
